codegen/templates/plugin/plugin.tmpl

```diff
@@ -38,13 +38,14 @@ func (p *Plugin) CallResource(ctx context.Context, req *backend.CallResourceRequ
 
 func New(namespace string, service Service) (*Plugin, error) {
 	p := &Plugin{
-	    router: router.NewJSONRouter(log.DefaultLogger),
+	    router: router.NewJSONRouter(),
 			namespace: namespace,
 	    service: service,
 	}
 
 	p.router.Use(
 		router.NewTracingMiddleware(otel.GetTracerProvider().Tracer("tracing-middleware")),
+		router.NewLoggingMiddleware(logging.DefaultLogger),
```

Comment:
Do you think it's better this logging middleware consumes the DefaultLogger or should we let `New` consume the logger used in `newInstanceFactory` and pass it here too?
From what I can see the `newInstanceFactory` might be a good place to set some additional fields from instance settings. Or do you think that `DefaultLogger` would be the one having those fields?

Reply:
In this case, I guess I mostly assumed that the default logger for the plugin would have all the expected information from the plugin init process.

Reply:
Is fine for me to assume that as the default logger can be updated in the `app.Manage`  instance Factory. 
The important thing here is that is clear to developers that they can do it 👍🏼